site/content/tutorial/07-lifecycle/02-ondestroy/app-a/Timer.svelte

```diff
@@ -0,0 +1,19 @@
+<script>
+	import { onInterval } from './utils.js';
+
+	export let callback;
+	export let interval = 1000;
+
+	onInterval(callback, interval);
+</script>
+
+<p>
+	This component executes a callback every 
+	{interval} millisecond{interval === 1 ? '' : 's'}
+</p>
+
+<style>
+	p { 
+		border: 1px solid blue;
```

Comment:
I might also add something like `padding: 5px;` here to make it look a bit nicer (and same below with the other blue border)

Reply:
```suggestion
		border: 1px solid blue;
		padding: 5px;
```